cmd/src/debug_comp.go

```diff
@@ -0,0 +1,243 @@
+package main
+
+import (
+	"archive/zip"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+
+	"golang.org/x/sync/errgroup"
+
+	"golang.org/x/sync/semaphore"
+
+	"github.com/sourcegraph/src-cli/internal/cmderrors"
+)
+
+func init() {
+	usage := `
+'src debug compose' invokes docker cli diagnostic commands targeting a containers that are members of a docker-compose network, 
+writing an archive file from their returns. 
+
+Usage:
+
+    src debug compose [command options]
+
+Flags:
+
+	-o			    Specify the name of the output zip archive.
+	--no-configs		Don't include Sourcegraph configuration json.
```

Comment:
I think you've mixed up tabs and spaces in these strings. If you're on vs code you can view whitespace characters to see if they're the same.